### PR TITLE
Feat/4 add dev warning

### LINKED ISSUE #4: Add Dev Warning
Add note that this environment is not suitable for production usage.

### PR DESCRIPTION
**Pull request by Xelef2000**
Added a Dev warning to USAGE.md and added a wrapper script for compose up and down which prints a dev warning.

### CODE REVIEW

File: USAGE.md [hunk 1]
```diff
@@ -183,7 +185,7 @@ Persist it to the file [`docker/terraform/vault-auth-kubernetes.tf`](./docker/te
 Run Terraform to provision Vault and Keycloak:
 ```bash
 sudo ${CONTAINER_RUNTIME}-compose rm terraform
-sudo ${CONTAINER_RUNTIME}-compose up -d terraform
+sudo start.sh terraform
```
Comment (Xelef2000) [inline USAGE.md]: This is wrong, I'll update it.

File: USAGE.md [hunk 2]
```diff
@@ -363,7 +365,7 @@ kubectl get sa vault-kv -n vault-app -o yaml | grep uid
...
```
Comment (Xelef2000) [inline USAGE.md]: This too.

**Review by ghost — APPROVED** [review]
I like it, thanks a lot. We might organize the cleanup in a similar way? I added a first draft for a cleanup script (reuses stop script) and structured the scripts by enumerating them.

**Comment by Xelef2000** [comment]
I will save the changes, delete this branch and create a new one since the initial commit in this PR is not signed.

**Comment by ghost** [comment]
Yes, noticed the strange behavior as well, was all blocked. Follow-up is in https://github.com/adfinis/vault-playground/pull/22

**Pull request closed without merging**